app/models/inventory_item.rb

```diff
@@ -1,2 +1,9 @@
 class InventoryItem < ActiveRecord::Base
+  using(:grossman)
+  after_initialize :init
```

Comment:
I am having a hard time figuring out how to work around this issue. Payroll seems to have the same methods here: https://github.com/westernmilling/payroll-app/blob/master/app/models/driver.rb